tests/functional/aws-node-sdk/package.json

```diff
@@ -20,7 +21,7 @@
   },
   "scripts": {
     "test-aws": "AWS_ON_AIR=true mocha",
-    "test": "mocha",
+    "test": "mocha ",
```

Comment:
The diff is explicit enough, right? It's only adding a space for no particular reason if you want to know.